docs/concepts/memap.md

```diff
@@ -64,20 +66,34 @@ The table above showed multiple memory sections.
 - ``.text``: is where the code application and constants are located in Flash.
 - ``.data``: nonzero initialized variables; allocated in both RAM and Flash memory (variables are copied from Flash to RAM at runtime).
 - ``.bss``: uninitialized data allocated in RAM, or variables initialized to zero.
-- ``Heap``: dynamic allocations in the Heap area in RAM (for example, used by ``malloc``). The maximum size value may be defined at build time.
-- ``Stack``: dynamic allocations in the Stack area in RAM (for example, used to store local data, temporary data when branching to a subroutine or context switch information). The maximum size value may be defined at build time.
+
+**Note**: This tool displays static RAM and Flash usage. The heap and stack space are not included in the report as are usually configured at run time, and it's likely to change from one application to another.
```

Comment:
Please change the second sentence to "The report does not include heap and stack space because they are usually configured at runtime and are likely to change from one application to another." for active voice.